documentation/bolt_installing.md

```diff
@@ -42,7 +42,16 @@ have installed:
   ```shell
   wget https://apt.puppet.com/puppet-tools-release-buster.deb
   sudo dpkg -i puppet-tools-release-buster.deb
-  sudo apt-get update 
+  sudo apt-get update
+  sudo apt-get install puppet-bolt
+  ```
+
+- _Debian 11_
+
+  ```shell
+  wget https://apt.puppet.com/puppet-tools-release-bullseye.deb
```

Comment:
This repo isn't currently available. We will need to release it prior to releasing Bolt, but should not do it now as the repo would be empty.

Reply:
Touched base with Eric, we can just make sure this repo is available before releasing.